File: Admin/src/model/Category.ts
class Category {
    public categoryId: number;
    public categoryName: string;
    public categoryIcon: string;
    public constructor(categoryId: number, categoryName: string, categoryIcon: string) {
        this.categoryId = categoryId;
        this.categoryName = categoryName;
        this.categoryIcon = categoryIcon;
    }
}

export default Category;

File: SellingElectronicWebsite/SellingElectronicWebsite/Entities/Category.cs
using System;
using System.Collections.Generic;

namespace SellingElectronicWebsite.Entities;

public partial class Category
{
    public int CategoryId { get; set; }

    public string? CategoryName { get; set; }

    public string? CategoryIcon { get; set; }

    public virtual ICollection<Product> Products { get; set; } = new List<Product>();
}
